### PR TITLE
docs: update vault kms integration doc

### PR DESCRIPTION
**Pull request by thotz**
**Description of your changes:**
correct the name of policy mentioned

**Which issue is resolved by this Pull Request:**
Resolves #

Signed-off-by: Jiffin Tony Thottan <jthottan@redhat.com>
**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

File: Documentation/ceph-cluster-crd.md [hunk 1]
```diff
@@ -1427,7 +1427,7 @@ capabilities = ["read"]
 You can write the policy like so and then create a token:
 
 ```console
```
Comment (thotz) [inline Documentation/ceph-cluster-crd.md:1429]: @leseb : I have two doubts
1.) do we really need 
```hcl
path "sys/mounts" {
capabilities = ["read"]
}
```
as part of this policy?

2.) Do we need to mention how encryption keys are created in the vault here?

Comment (thotz) [reply Documentation/ceph-cluster-crd.md:1429]: I understood part, rooks internally create an encryption key for the osd encryption

Comment (leseb) [reply Documentation/ceph-cluster-crd.md:1429]: > @leseb : I have two doubts
> 1.) do we really need
> 
> ```terraform
> path "sys/mounts" {
> capabilities = ["read"]
> }
> ```
> 
> as part of this policy?

Yes we do.

> 
> 2.) Do we need to mention how encryption keys are created in the vault here?

Comment (leseb) [reply Documentation/ceph-cluster-crd.md:1429]: > I guess I understood 2nd question, rooks internally create an encryption key for the osd.
> So creating encryption key via `vault kv put ` is not required to mention here

Correct.

**Review by leseb — APPROVED** [review]

**Pull request merged**